tests/integration/cli.py

```diff
@@ -8,32 +9,87 @@
 
 
 PROJECT1 = 'foo'
-PROJECT2 = 'bar'
-
-
-def test_add_list_delete_projects():
-    """ Test the CRUD operations on projects from the CLI.
-
-    The project list does not need to be empty before running the test."""
-    output = subprocess.check_output(['hil', 'list_projects'])
-    projects = output.split(" ")
-    projects.remove('\n')
-    assert PROJECT1 not in projects
-    assert PROJECT2 not in projects
-    size = len(projects)
-
-    subprocess.check_call(['hil', 'project_create', PROJECT1])
-    output = subprocess.check_output(['hil', 'list_projects'])
-    output = output.strip('\n')
-    projects = output.split(" ")
-    assert PROJECT1 in projects
-    assert PROJECT2 not in projects
-    assert len(projects) == size + 1
-
-    subprocess.check_call(['hil', 'project_delete', PROJECT1])
-    output = subprocess.check_output(['hil', 'list_projects'])
-    projects = output.split(" ")
-    projects.remove('\n')
-    assert PROJECT1 not in projects
-    assert PROJECT2 not in projects
-    assert len(projects) == size
+NODE1 = 'dell-1'
+NIC1 = 'eth1'
+
+SWITCH1 = 'mock-switch1'
+SWITCH2 = 'mock-switch2'
+PORT = 'gi1/0/1'
+
+
+def hil(*args):
+    """Convenience function that calls the hil command line tool with
+    the given arguments.
+    """
+    args = " ".join(args)
+    args = args.split(" ")
```

Comment:
Just for the convenience, so that I could quickly type the commands in the tests.
`hil('node metadata add', NODE1, 'metadata-label metadata-value')` vs `hil('node', 'metadata', 'add', NODE1, 'metadata-label', 'metadata-value')`. If you think it's not the right way to do it, I don't mind changing it.